### DISCUSSION TITLE
Wallet needs to be fully synced before sending transactions

### DISCUSSION THREAD

**Discussion by TSchaefer78** [Chia-Network/chia-blockchain#8089] [Support]
Hi guys,

I have the problem that I cannot execute a transaction. I get the following message: "Wallet needs to be fully synced before sending transactions".
My wallet is completely synchronized.
Why can't I make a transaction?

Please help
Thanks, Tom

**Comment by keliew** [comment]
You mean you `node` is synced? Or your `wallet` is synced? Two different things.

**Comment by BrandtH22** [comment]
@TSchaefer78 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
